### PR TITLE
cockpit: Introduce template react components for all configuration subpages

### PR DESCRIPTION
**Pull request by KKoukiou**
Also start implementing the language selection intro menu and the root
password subpage which can be now used as a code reference.

@patternfly/react-core was updated to the latest available so as to be
able to use the latest features implemented.

react/prop-types rule was disabled in eslint, as I find that PropTypes
add a lot of code and little help.

Lastly introduce a new test file for the root account subpage, and since
now we have more than one (2!!) test files, let's use the
test/common/run-test wrapper for running the tests in CI.

### CODE REVIEW

**Comment by KKoukiou** [comment]
Current state with this PR:

Intro page to select language for the installation:
Screen Shot 2022-01-07 at 16 07 45

Summary page:
Screen Shot 2022-01-07 at 16 05 53

Root accounts sub-page:
Screen Shot 2022-01-07 at 16 06 05

**Comment by KKoukiou** [comment]
Need to check the tests - pls odn't merge.

File: pyanaconda/ui/cockpit/anaconda-webui/src/InstallationDestination.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2022 Red Hat, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
```
Comment (M4rtinK) [inline pyanaconda/ui/cockpit/anaconda-webui/src/InstallationDestination.jsx:5]: I'm thinking we should discuss what license we want to use for the Web UI files - traditionally we have used GPL 2+ for almost everything in Anaconda with LGPL 2+ in just a couple places, mostly some utility scripts.

**Review by M4rtinK — APPROVED** [review]
Looks good to me, thanks a lot! :)

But more importantly - WOW! Like really! :)

Considering what a pain it is to implement even minor things in GTK/Glade, this is quite something! Rapid prototyping in action! :)

File: pyanaconda/ui/cockpit/anaconda-webui/src/InstallationDestination.jsx (hunk 1 above)
Comment (KKoukiou) [reply pyanaconda/ui/cockpit/anaconda-webui/src/InstallationDestination.jsx:5]: Indeed. wednesday topic it is.

**Pull request merged**